src/Table/SynapseTableReflection.php

```diff
@@ -316,18 +315,84 @@ public function getTableDefinition(): TableDefinitionInterface
                 $this->getTableDistributionColumnsNames()
             ),
             new TableIndexDefinition(
-                $this->getTableIndexType(),
-                []
+                $this->getTableIndex(),
+                $this->getTableIndexColumnsNames()
             )
         );
     }
 
     /**
-     * @todo find the way how to get right index from synapse
      * @return TableIndexDefinition::TABLE_INDEX_TYPE_*
      */
-    public function getTableIndexType(): string
```

Comment:
Bacha toto asi moze niekto pouzivat ked pouziva tu libku ale v connection to nieje pouzite

Reply:
jop to sem udělal naschvál aby to bylo stejné jak distribution metoda. Řekl bych, že to používáme enom my snad :) Napíšu to do release notes.